src/main/java/com/google/devtools/build/lib/remote/http/HttpDownloadHandler.java

```diff
@@ -51,6 +45,10 @@ final class HttpDownloadHandler extends AbstractHttpHandler<HttpObject> {
   private long contentLength = -1;
   /** the path header in the http request */
   private String path;
+  /** the offset at which to download */
+  private long offset;
+  /** the bytes to skip in a full or chunked response */
+  private OptionalInt skipBytes;
```

Comment:
What is the reason for having both `offset` and `skipBytes` as fields? Could the code be simplified by removing the `offset` field and instead initialize `skipBytes` where `offset` is currently initialized?

Reply:
True, one field is enough. I've dropped `offset` in favor of `skipBytes`.